src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidator.java

```diff
@@ -15,7 +15,7 @@
 public class RemoveValidator {
 
     public Errors validate(RemoveDto removeDto, Errors errors, String loggingContext) {
-        if (removeDto.getIsNotProprietorOfLand() != null && removeDto.getIsNotProprietorOfLand() == false) {
+        if (removeDto.getIsNotProprietorOfLand() != null) {
```

Comment:
We can't remove `&& removeDto.getIsNotProprietorOfLand() == false` that's the key check this validator is performing.

Fix for the code smell would be to change it too:

`&& !removeDto.getIsNotProprietorOfLand()`

I think, but tests would confirm that.

Reply:
code smell addressed